main/src/main/java/com/sedmelluq/discord/lavaplayer/container/xm/XmTrackProvider.java

```diff
@@ -0,0 +1,46 @@
+package com.sedmelluq.discord.lavaplayer.container.xm;
+
+import com.sedmelluq.discord.lavaplayer.filter.AudioPipeline;
+import com.sedmelluq.discord.lavaplayer.filter.AudioPipelineFactory;
+import com.sedmelluq.discord.lavaplayer.filter.PcmFormat;
+import com.sedmelluq.discord.lavaplayer.track.playback.AudioProcessingContext;
+import ibxm.IBXM;
+
+public class XmTrackProvider {
+    private int BLOCKS_IN_BUFFER = 2048;
+    private final IBXM ibxm;
+    private final AudioPipeline downstream;
+
+    public XmTrackProvider(AudioProcessingContext context, IBXM ibxm) {
+        this.ibxm = ibxm;
+        this.downstream = AudioPipelineFactory.create(context, new PcmFormat(2, 44100));
+        BLOCKS_IN_BUFFER = ibxm.getMixBufferLength();
+    }
+
+    public void seekToTimecode(long timecode) {
```

Comment:
Just throw away the method for now (can use `null` instead where it's referenced).

Reply:
I've implemented that but it's not committed